### PR TITLE
adduser, translation-templates/*: add Finnish translation

### PR DESCRIPTION
**Pull request by armanatory**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by armanatory** [comment]
> {{path/to/home}}/{{path/to/shell}} and {{username}} could also be translated. See the [common arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md?rgh-link-date=2023-10-26T18%3A15%3A43Z) for reference. Feel free to add Finnish to the table.

Thanks for your feedback and review, I added that as well.

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for adding a new language 🚀. The placeholders in `adduser` still seems to be untranslated so feel free to translated it.

I would like to request you to add Finnish translation to the remaining translation template files.

**Comment by armanatory** [comment]
> adduser: add Finnish translation, translation-templates: add Finnish translation

Thanks for the review and comment, those are also now updated.

**Comment by armanatory** [comment]
> Welcome to tldr and thanks for adding a new language 🚀. The placeholders in `adduser` still seem to be untranslated so feel free to translate it (as suggested above).
> 
> I would like to request you to add Finnish translation to the remaining translation template files too similar to the common arguments file. The files in question are [alias-pages](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md) & [subcommand-mention](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md).

Thanks for the review and comment, those are also now updated.

File: contributing-guides/translation-templates/alias-pages.md [hunk 1]
```diff
@@ -185,6 +186,21 @@ The templates can be changed when necessary.
...
+> Tämä komento on `example`:n alias.
+
+- Katso alkuperäisen komennon dokumentaatiossa:
+
+`tldr example`
+```
+
+---
+
+
```
Comment (kbdharun) [inline contributing-guides/translation-templates/alias-pages.md:203]:
```suggestion
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kbdharun** [comment]
**Update**: Tested the PDF (https://tldr.sh/assets/tldr-book-fi.pdf) for the new language and the rendering looks fine 🎉

**Pull request merged**